components/Workspaces/WorkspaceLayout.tsx

```diff
@@ -11,9 +11,10 @@ interface WorkspaceLayoutProps {
   workspaceId: string;
   banner?: React.ReactNode;
   children: React.ReactNode;
+  footer?: React.ReactNode;
```

Comment:
The footer is optional (for now at least), because not all pages that use the workspaces layout need it.

Longterm, the update/create CTAs for insight pages should live in the footer as well @isabensusan to keep things uniform.